SPECS/kernel/kernel.spec

```diff
@@ -452,6 +467,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %endif
 
 %changelog
+* Mon Feb 22 2021 Thomas Crain <thcrain@microsoft.com> - 5.10.13.1-2
+- Add configs for speakup and uinput drivers
```

Comment:
Suggestion: maybe also mention the new subpackage name somewhere in the changelog.

Reply:
Fixed, thanks!